src/pages/Home/index.js

```diff
@@ -70,7 +78,8 @@ export default function Home() {
                         l'envoi, ces données sont effacées au bout d'un jour.
                         Pour en savoir plus sur la gestion des données
                         personnelles et pour exercer vos droits, veuillez vous
-                        reporter à la page <Link to="/donnees-personnelles">
+                        reporter à la page{' '}
```

Comment:
Whis this added space, the "Données personnelles" link disappears

![image](https://user-images.githubusercontent.com/50984435/80318773-b6700080-880c-11ea-91ae-de849e0ebba3.png)